File: Programmers/Lv0/삼각형의완성조건2.py
def solution(sides):
    _set = set()
    for i in range(max(sides) - min(sides) + 1, max(sides) + 1):
        _set.add(i)
    for j in range(max(sides) + 1, max(sides) + min(sides)):
        _set.add(j)
    return len(_set)

File: Programmers/Lv0/삼각형의완성조건2.cpp
#include <string>
#include <vector>
#include <set>
#include <algorithm>
using namespace std;

int solution(vector<int> sides) {
    set<int> _set;
    auto _max = max_element(sides.begin(), sides.end());
    auto _min = min_element(sides.begin(), sides.end());
    for (int i = *_max - *_min + 1; i < *_max + 1; i++)
        _set.insert(i);
    for (int j = *_max + 1; j < *_max + *_min; j++)
        _set.insert(j);
    return _set.size();
}